### PR TITLE
WIP : Supabase UI testing components

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

Adds a basic UI library using Storybook covering Buttons and Modals.
Other UI elements are a work in progress.
Primarily merging into master to test components locally.

### CODE REVIEW

File: supabase-ui/src/components/SideOver/SideOver.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { action, actions } from '@storybook/addon-actions'
+
+import Transition from './../../lib/Transition'
+import { Button } from './../../index'
+
+const SideOver = ({
```
Comment (kiwicopple) [inline supabase-ui/src/components/SideOver/SideOver.js:9]: "SideOver" is a new one for me, should this be "SlideOver"  (verb) or "SidePanel" (noun)?  

personal preference would be to use nouns for component and verbs for interactions (Transition etc) -  more consistent with our Dashboard

File: supabase-ui/src/index.js [hunk 2]
```diff
@@ -8,3 +8,12 @@ export * from './components/Input/index'
...
 export * from './components/Textarea/index'
+
+// OVERLAYS
+
+export * from './components/Modal/index'
+export * from './components/SideOver/index'
+
+// UTILITIES
+
+export * from './components/Transition/index'
```
Comment (kiwicopple) [inline supabase-ui/src/index.js:19]: Great, nice to move this one in here too - would be cool to have a few little interactions we can use across our apps

**Review by kiwicopple — APPROVED** [review]

File: supabase-ui/src/components/SideOver/SideOver.js (hunk 1 above)
Comment (MildTomato) [reply supabase-ui/src/components/SideOver/SideOver.js:9]: I think it was a name from tailwind.
we're not going to use this at the minute, as we need nested SideOvers - but kept it in to update later.

Comment (kiwicopple) [reply supabase-ui/src/components/SideOver/SideOver.js:9]: You sure it wasnt' "Slide overs"? https://tailwindui.com/components/application-ui/overlays/slide-overs

File: supabase-ui/package-lock.json [hunk 3]
```diff
@@ -1237,7 +1237,10 @@
       "version": "7.12.5",
       "resolved": "https://registry.npmjs.org/@babel/runtime/-/runtime-7.12.5.tgz",
       "integrity": "sha512-plcc+hbExy3McchJCEQG3knOsuh3HH+Prx1P6cLIkET/0dLuQDEnrT+s27Axgc9bqfsmNUNHfscgMUdBpC9xfg==",
+<<<<<<< HEAD
```
Comment (phamhieu) [inline supabase-ui/package-lock.json:1240]: there are merge conflicts in this file. You should remove it, run `npm install` to re-create it @MildTomato

**Review by phamhieu — CHANGES_REQUESTED** [review]

**Comment by MildTomato** [comment]
@phamhieu this PR is cancelled. moving to new repo

**Comment by MildTomato** [comment]
cancelling this PR as its outdated

UI library now moved to https://github.com/supabase/ui

**Pull request closed without merging**